lib/modules/manager/bundler/artifacts.ts

```diff
@@ -43,8 +43,24 @@ function buildBundleHostVariable(hostRule: HostRule): Record<string, string> {
   };
 }
 
+const resolvedPkgRegex = regEx(
+  /(?<pkg>\S+)(?:\s*\([^)]+\)\s*)? was resolved to/g
+);
+
+function getResolvedPackages(input: string): string[] {
+  const result: string[] = [];
+  let resolveMatchGroups = resolvedPkgRegex.exec(input)?.groups;
```

Comment:
i think this will cause issues if you pass exactly the same string twice to this function, the second time it won't find anything.

can we split the string by newlines instead and iterate over?

the big issue is: do not reuse cached regex, as it stores state